style.css

```diff
@@ -0,0 +1,82 @@
+* {
+  font-family: Poppins, sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+.main-block {
+  background-color: #e5e5e5;
+}
+
+.head-menu {
+  background-color: #fff;
+}
+
+.nav-mobile {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  color: #6070ff;
+  text-decoration: none;
+  margin-top: 6.3vh;
+  margin-left: 2.8vh;
+  font-weight: 700;
+  font-size: 2vh;
+}
+
+nav > .fa-bars {
+  margin-top: 6.3vh;
+  margin-right: 4.2vh;
+  color: #6070ff;
+}
+
+.home-page {
+  background-image: url("./Images/header-shapes-mobile@2x.png");
+  background-repeat: no-repeat;
+  background-color: #fff;
+  height: 80vh;
+  border-bottom-left-radius: 9vh;
+}
```

Comment:
I believe you noticed that the background image is not responsive to the screen size. You can fix this.

- [ ] Make sure to apply `background-size: 100vw;` to the `.home-page` class in order to have a background image of 100% of the view width.

Reply:
Thanks for the suggestion. I have done the required changes.